modules/imgproc/src/drawing.cpp

```diff
@@ -39,7 +39,7 @@
 //
 //M*/
 #include "precomp.hpp"
-
+#include <iostream>
```

Comment:
Why do we need that?

Reply:
We don't need (forgotten test)
delete in next commit